paper/basic_training.tex

```diff
@@ -568,73 +568,104 @@ \subsection{Main steps of a molecular dynamics simulation}
 
-At the end of energy minimization, it is important to achieve a system configuration with small enough forces that the desired timestep will allow numerical integration of the equations of motion without overly large displacements (see \citet{LeachBook}, section 7.3.4).
-Such a configuration is a suitable starting point for molecular dynamics techniques.
-However, this only represents a static set of positions, while the propagation of dynamics also requires a set of starting velocities.
-These may be assigned in a variety of ways, but are usually randomly assigned to atoms such that the correct Maxwell-Boltzmann distribution at the desired temperature is achieved.
+\subsubsection{Assignment of velocities}
+Minimization ideally takes us to a state from which we can begin numerical integration of the equations of motion without overly large displacements (see \citet{LeachBook}, section 7.3.4); however, to begin a simulation, we need not just positions but also velocities.
+Minimization, however, provides only a final set of positions.
+Thus, starting velocities must be assigned; usually this is done by assigning random initial velocities to atoms in a way such that the correct Maxwell-Boltzmann distribution at the desired temperature is achieved as a starting point.
+
+In some cases, we seek to obtain multiple separate and independent simulations of different instances or realizations of a particular system to assess error, collect better statistics, or help gauge dependence of results on the starting structure.
+It is worth noting that even very small differences in initial configuration, such as even a difference in position of a single atom, lead to exponential divergence of the time evolution of the system~\cite{allen_computer_2017}, meaning that simply running different simulations starting with different initial velocities will lead to dramatically different time evolution over long enough times.
+Still, an even better way to generate independent realizations is to begin with different starting configurations, such as different conformations of the molecule(s) being simulated, as this leads to behavior which is immediately different.
+
+\subsubsection{Equilibration}
+
+Ultimately, we usually seek to run a simulation in a particular thermodynamic ensemble (e.g. the NVE or NVT ensemble) at a particular state point (e.g. target energy, temperature, and pressure) and collect data for analysis which is appropriate for those conditions and not biased depending on our starting conditions/configuration.
+This means that usually we need to invest simulation time in bringing the system to the appropriate state point and allowing it to essentially forget about its history and reach equilibrium (or pseudo-equilibrium -- for some systems, such as biomolecular systems, reaching true equilibrium may be impractical) before we begin retaining data for analysis.
+
```

Comment:
Well, the system could still be at equilibrium (say, rigorously in the NVT ensemble), but _sampling_ the equilibrium distribution in such a way that we have good statistics that are not biased by the initial configuration is a different story.  I think you're referring to the later, right?  Even configurations with very high energy _could_ be sampled at equilibrium, just with low probability.  I agree we need to make these issues clear, but maybe this is touching more on the sampling and convergence document?

Reply:
I think (but perhaps I'm missing something) that we can't say we're rigorously in the NVT (for example) ensemble until we've sufficiently relaxed away from our initial configuration. For example, it might look like the temperature has stabilized but if, say, a protein is still undergoing a large conformational change then that's going to be leading to transient behavior that we need to have die out before we're really at equilibrium in that ensemble (e.g. the thermostat will be needing to dump lots of energy into/out of the system in such cases). 

To put it another way, your OVERALL temperature might appear constant (leading you to think that you've equilibrated to NVT) but the temperature of PARTS of your system might be hotter/colder (e.g. your protein is moving faster because it's re-folding, converting potential energy to kinetic energy in the process) and thus you've not yet equilibrated.

So what I'm saying is that monitoring "whole system" quantities like temperature doesn't ensure you've equilibrated sufficiently; the components of your system also need to have equilibrated.

Convergence is separate and relates to whether we've sampled enough to obtain good estimates of properties. But until we've equilibrated it's not worth thinking about convergence.